Rules/CommonScripts/DefaultVotes.as

```diff
@@ -578,7 +600,19 @@ void onMainMenuCreated(CRules@ this, CContextMenu@ menu)
 	//nextmap menu
 	if (getSecurity().checkAccess_Feature(me, "map_vote"))
 	{
-		if (g_lastNextmapCounter < 60 * getTicksASecond()*required_minutes_nextmap
+		if (duplicatePlayer)
+		{
+			Menu::addInfoBox(
+				mapmenu,
+				getTranslatedString("Can't Start Vote"),
+				getTranslatedString(
+					"Voting for next map\n" +
+					"is not allowed when playing\n" +
+					"with a duplicate instance of KAG"
```

Comment:
Might be worth telling the user to rejoin the server. Most the time people rejoin a server after they lag out (before their last player gets removed), causing a ~ to appear. In that case those people might not be aware they are a duplicate.